tests/test_converter.py

```diff
@@ -43,7 +44,9 @@ def existing_dataset(tmp_path_factory) -> pathlib.Path:
     tmp_path = tmp_path_factory.mktemp("dataset")
     csv_content = f"{sample_csv_headers}\n{sample_csv_data}"
     csv_data = pd.read_csv(StringIO(csv_content))
-    convert_colorizer_data(csv_data, tmp_path, output_format=DataFileType.JSON)
+    convert_colorizer_data(
+        csv_data, tmp_path, output_format=DataFileType.JSON, image_column="File Path"
```

Comment:
image_column became optional, so I manually added it to a few of the tests here.